celery/bin/multi.py

```diff
@@ -288,7 +288,7 @@ def _handle_reserved_options(self, argv):
 
     @splash
     @using_cluster
-    def start(self, cluster):
+    def start(self, cluster, **kwargs):
```

Comment:
thanks for the PR, can you please add some test for the **kwargs part?

Reply:
No need for CLI tests right now.
I'll work on them later.